.github/workflows/release-fleet.yml

```diff
@@ -51,6 +53,15 @@ jobs:
           python -m pip install --upgrade pip
           pip install .[dev]
 
+      - name: Bump prebuilt rules package version
+        env:
+          PACKAGE_MATURITY: "${{github.event.inputs.package_maturity}}"
```

Comment:
how do we handle _major_ or non standard bumps? Could expose it as an unrequired input, where if undefined, it uses this

Reply:
We will run the workflow from a specific branch where `packages.yml` is different at each branch but essentially the stack versions match that branch.

Therefore if we run from 9.0 branch, the registry version will already be `9.0.0.beta.1` and kibana stack version is `^9.0.0`, therefore we are only concerned about the beta and ga package versions for the patch version.

Workflow wise:

branch: `9.0`
package version default: `9.0.0-beta.0`
determined release fleet package version `9.0.0-beta.1`

From `integrations.py` we use `find_latest_integration_version()` which would return, for example, `8.10.1-beta.2` as the latest version in EPR.

In [L61](https://github.com/elastic/detection-rules/blob/315e3cf1b7215168cf0e5955cc6a7999366cfbee/.github/workflows/release-fleet.yml#L61) we bump package versions based on the latest (`8.10.1-beta.2`) and maturity `beta`. In the command the following should catch such examples for a major or minor that does not have an existing package.

```
# if an existing minor or major does not have a package, bump from the last
# example is 8.10.0-beta.1 is last, but on 9.0.0 major
# example is 8.10.0-beta.1 is last, but on 8.11.0 minor
if latest_patch_release_ver.minor != pkg_kibana_ver.minor:
    latest_patch_release_ver = latest_patch_release_ver.bump_minor()
if latest_patch_release_ver.major != pkg_kibana_ver.major:
    latest_patch_release_ver = latest_patch_release_ver.bump_major()
```